doc/api_assets/style.css

```diff
@@ -207,6 +213,12 @@ header h1 {
   padding-top: 1em;
 }
 
+#apicontent .line {
+  width: calc(50% - 1em);
```

Comment:
There was something about it resulting in more and multiple passes during reflow, but I think that is outdated. Scratch the above then. Thx